service.libraryautoupdate/resources/__init__.py

```diff
@@ -0,0 +1 @@
+# Dummy file to make this directory a package.
```

Comment:
shouldn't be needed anymore. Please consider dropping for a next update

Reply:
Will do! Right now everything is still python 2 and 3 compatible. With the next Matrix update I'm going to strip out the Python 2 stuff.